data/statuses.js

```diff
@@ -479,7 +479,7 @@ let BattleStatuses = {
 		},
 		onWeatherModifyDamage(damage, attacker, defender, move) {
 			if (move.type === 'Water') {
-				if (defender.item === "utilityumbrella") return;
+				if (defender.hasItem('utilityumbrella')) return;
```

Comment:
The mechanics report says that the nerf is also suppressed, so you could just move the check to the beginning of the function.